### PR TITLE
Fix typos and wrong links in SimpleEvents

### PR DESCRIPTION
**Pull request by Matocolotoe**
### Description

Fixes typos and other links leading to an invalid page in docs

### CODE REVIEW

**Comment by Blueyescat** [comment]
#2206

**Comment by Matocolotoe** [comment]
Actually added other things

File: src/main/java/ch/njol/skript/events/SimpleEvents.java [hunk 1]
```diff
@@ -307,34 +307,34 @@
...
+						"Use the <a href='#damage'>damage event</a> with a <a href='conditions.html#CondIsSet'>check</a> for a <a href='expressions/#ExprEntity'>projectile</a> " +
+								"to be able to use the <a href='expressions.html#ExprAttacked'>entity that got hit</a> in the case when the projectile hit a living entity.",
 						"A damage event will even be fired if the damage is 0, e.g. when throwing snowballs at non-nether mobs.")
 				.examples("on projectile hit:",
 						"\tevent-projectile is arrow",
 						"\tdelete event-projectile")
 				.since("1.0");
 		Skript.registerEvent("Shoot", SimpleEvent.class, ProjectileLaunchEvent.class, "[projectile] shoot")
-				.description("Called whenever a <a href='../classes.html#projectile'>projectile</a> is shot. Use the <a href='../expressions/#ExprShooter'>shooter expression</a> to get who shot the projectile.")
+				.description("Called whenever a <a href='classes.html#projectile'>projectile</a> is shot. Use the <a href='expressions/#ExprShooter'>shooter expression</a> to get who shot the projectile.")
```
Comment (bensku) [inline src/main/java/ch/njol/skript/events/SimpleEvents.java]: This is still broken link.

Comment (Matocolotoe) [reply src/main/java/ch/njol/skript/events/SimpleEvents.java]: Fixed in next commit :)

File: src/main/java/ch/njol/skript/events/SimpleEvents.java [hunk 2]
```diff
@@ -107,7 +107,7 @@
 	static {
 		Skript.registerEvent("Can Build Check", SimpleEvent.class, BlockCanBuildEvent.class, "[block] can build check")
 				.description("Called when a player rightclicks on a block while holding a block or a placeable item. You can either cancel the event to prevent the block from being built, or uncancel it to allow it.",
-						"Please note that the <a href='expressions.html#ExprDurability'>data value</a> of the block to be placed is not available in this event, only its <a href='expressions/#ExprIdOf'>ID</a>.")
+						"Please note that the <a href='expressions.html#ExprDurability'>data value</a> of the block to be placed is not available in this event, only its <a href='expressions#ExprIdOf'>ID</a>.")
```
Comment (bensku) [inline src/main/java/ch/njol/skript/events/SimpleEvents.java]: Still broken link.

Comment (Matocolotoe) [reply src/main/java/ch/njol/skript/events/SimpleEvents.java]: Fixed

**Pull request merged**